src/outputs/output_utils.hpp

```diff
@@ -135,6 +136,8 @@ struct VarInfo {
           bool is_vector, const IndexShape &cellbounds)
       : label(label), num_components(num_components), nx_(nx),
         tensor_rank(metadata.Shape().size()), where(metadata.Where()),
+        is_mem_aligned(metadata.IsSet(Metadata::CellMemAligned) &&
+                       metadata.IsSet(Metadata::Flux)),
```

Comment:
Is this only tied to flux fields (based on this code) or more general for `CellMemAligned` (based on the comment in line 60 above)?

I tried to make a connection to the metadata doc (`By default, a flux field for a cell-centered field is built with Metadata::CellMemAligned flag set for backwards compatability.`) but I guess I'm not entirely aware of what `CellMemAligned` actually means.

Reply:
`CellMemAligned` is for face fields only, I think. It basically drops 1 ghost zone on the upper faces of the block so that the memory footprint for each face is the same as for cells. This is how fluxes worked in Athena++ before @lroberts36 turned fluxes into face-centered vars and unified that machinery.

I don't think you can add `CellMemAligned` to edge- or node-centered data.

Reply:
> I don't think you can add `CellMemAligned` to edge- or node-centered data.

> You _can_ make a non-flux face var `CellMemAligned` if you want to. And there might be reasons to do so.

It does work with face-, edge-, and node-centered data and doesn't require `Metadata::Flux`. But if you automatically make fluxes associated with a field, the associated flux field has `Metadata::CellMemAligned` and `Metadata::Flux` set automatically for backwards compatibility.